### PR TITLE
PRL-4443:Added changes to show and hide events in pending status

### PR DESCRIPTION
**Pull request by khurshedk**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/PRL-4443

### Change description ###
1. Remove the events - Review documents, Edit n serve order, Attach scanned docs
2. Retain the events - Add case note, Create flag, Manage flag
3. Judges must not be able to view or access CA/DA applications in Pending status

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[X ] No
```

### CODE REVIEW

File: definitions/private-law/json/CaseEvent/CaseEvent.json [hunk 1]
```diff
@@ -816,7 +816,7 @@
     "Name": "Attach scanned docs",
     "Description": "Attach scanned docs",
     "DisplayOrder": 24,
-    "PreConditionState(s)": "*",
+    "PreConditionState(s)": "AWAITING_SUBMISSION_TO_HMCTS;AWAITING_RESUBMISSION_TO_HMCTS;SUBMITTED_PAID;CASE_ISSUED;JUDICIAL_REVIEW;ALL_FINAL_ORDERS_ISSUED;PREPARE_FOR_HEARING_CONDUCT_HEARING;DECISION_OUTCOME",
```
Comment (yogendraU) [inline definitions/private-law/json/CaseEvent/CaseEvent.json:819]: Is this event required for AWAITING_SUBMISSION_TO_HMCTS;AWAITING_RESUBMISSION_TO_HMCTS?

File: definitions/private-law/json/CaseEvent/CaseEvent.json [hunk 2]
```diff
@@ -1686,7 +1686,7 @@
     "Name": "Review documents",
     "Description": "Review documents uploaded",
     "DisplayOrder": 48,
-    "PreConditionState(s)": "*",
+    "PreConditionState(s)": "AWAITING_SUBMISSION_TO_HMCTS;AWAITING_RESUBMISSION_TO_HMCTS;SUBMITTED_PAID;CASE_ISSUED;JUDICIAL_REVIEW;ALL_FINAL_ORDERS_ISSUED;PREPARE_FOR_HEARING_CONDUCT_HEARING;DECISION_OUTCOME",
```
Comment (yogendraU) [inline definitions/private-law/json/CaseEvent/CaseEvent.json:1689]: Is this event required for AWAITING_SUBMISSION_TO_HMCTS;AWAITING_RESUBMISSION_TO_HMCTS?

File: definitions/private-law/json/CaseEvent/Order/CaseEvent-nonprod.json [hunk 3]
```diff
@@ -42,7 +42,7 @@
     "CallBackURLAboutToStartEvent": "${CCD_DEF_COS_URL}/populate-draft-order-dropdown",
     "CallBackURLAboutToSubmitEvent": "${CCD_DEF_COS_URL}/judge-or-admin-edit-approve/about-to-submit",
     "CallBackURLSubmittedEvent": "${CCD_DEF_COS_URL}/judge-or-admin-edit-approve/submitted",
-    "PreConditionState(s)": "*",
+    "PreConditionState(s)": "AWAITING_SUBMISSION_TO_HMCTS;AWAITING_RESUBMISSION_TO_HMCTS;SUBMITTED_PAID;CASE_ISSUED;JUDICIAL_REVIEW;ALL_FINAL_ORDERS_ISSUED;PREPARE_FOR_HEARING_CONDUCT_HEARING;DECISION_OUTCOME",
```
Comment (yogendraU) [inline definitions/private-law/json/CaseEvent/Order/CaseEvent-nonprod.json:45]: Is this event required for AWAITING_SUBMISSION_TO_HMCTS;AWAITING_RESUBMISSION_TO_HMCTS?

File: definitions/private-law/json/CaseEvent/Order/CaseEvent-prod.json [hunk 4]
```diff
@@ -42,7 +42,7 @@
     "CallBackURLAboutToStartEvent": "${CCD_DEF_COS_URL}/populate-draft-order-dropdown",
     "CallBackURLAboutToSubmitEvent": "${CCD_DEF_COS_URL}/judge-or-admin-edit-approve/about-to-submit",
     "CallBackURLSubmittedEvent": "${CCD_DEF_COS_URL}/judge-or-admin-edit-approve/submitted",
-    "PreConditionState(s)": "*",
+    "PreConditionState(s)": "AWAITING_SUBMISSION_TO_HMCTS;AWAITING_RESUBMISSION_TO_HMCTS;SUBMITTED_PAID;CASE_ISSUED;JUDICIAL_REVIEW;ALL_FINAL_ORDERS_ISSUED;PREPARE_FOR_HEARING_CONDUCT_HEARING;DECISION_OUTCOME",
```
Comment (yogendraU) [inline definitions/private-law/json/CaseEvent/Order/CaseEvent-prod.json:45]: Is this event required for AWAITING_SUBMISSION_TO_HMCTS;AWAITING_RESUBMISSION_TO_HMCTS?

**Review by yogendraU — CHANGES_REQUESTED** [review]

File: definitions/private-law/json/CaseEvent/Order/CaseEvent-prod.json (hunk 4 above)
Comment (khurshedk) [reply definitions/private-law/json/CaseEvent/Order/CaseEvent-prod.json]: @yogendraU  removed AWAITING_SUBMISSION_TO_HMCTS;AWAITING_RESUBMISSION_TO_HMCTS as per below confluence page
https://tools.hmcts.net/confluence/pages/viewpage.action?pageId=1721438715

File: definitions/private-law/json/CaseEvent/Order/CaseEvent-nonprod.json (hunk 3 above)
Comment (khurshedk) [reply definitions/private-law/json/CaseEvent/Order/CaseEvent-nonprod.json]: @yogendraU  removed AWAITING_SUBMISSION_TO_HMCTS;AWAITING_RESUBMISSION_TO_HMCTS as per below confluence page
https://tools.hmcts.net/confluence/pages/viewpage.action?pageId=1721438715

File: definitions/private-law/json/CaseEvent/CaseEvent.json (hunk 2 above)
Comment (khurshedk) [reply definitions/private-law/json/CaseEvent/CaseEvent.json]: @yogendraU removed AWAITING_SUBMISSION_TO_HMCTS though this event is required in AWAITING_RESUBMISSION_TO_HMCTS as per 
https://tools.hmcts.net/confluence/pages/viewpage.action?pageId=1721438715

File: definitions/private-law/json/CaseEvent/CaseEvent.json (hunk 1 above)
Comment (khurshedk) [reply definitions/private-law/json/CaseEvent/CaseEvent.json]: @yogendraU removed AWAITING_SUBMISSION_TO_HMCTS though this event is required in AWAITING_RESUBMISSION_TO_HMCTS as per 
https://tools.hmcts.net/confluence/pages/viewpage.action?pageId=1721438715

**Review by yogendraU — APPROVED** [review]

**Review by AlokDatta — APPROVED** [review]

**Pull request merged**